api/lib/application/error-manager.js

```diff
@@ -20,6 +20,9 @@ function _mapToHttpError(error) {
     return error;
   }
 
+  if (error instanceof DomainErrors.CampaignAlreadyArchivedError) {
+    return new HttpErrors.PreconditionFailedError('Campaign is archived.');
```

Comment:
Je pense que le message devrait être porté par la classe CampaignAlreadyArchivedError. Le constructeur de CampaignAlreadyArchivedError devrait définir le message directement avec 'Campaign is archived.' sans  utilisé de paramètre par défaut.

```javascript
constructor() {
    super('Cette campagne est déjà archivée.');
  }
```
On utilise cette erreur à une seul endroit et je pense que ça ne changera pas. On a pas besoin de message par défaut et on en aura probablement jamais besoin.

Reply:
J'en profite je pose ça ici, dans ce fichier il y a des erreurs en anglais et en français. Du coup pourquoi la notre est en anglais ?